armoryengine/Networking.py

```diff
@@ -21,14 +21,15 @@
 
```

Comment:
well it is needed for segwit compatibility to work. Should I reflect these changes into those files?